### PR TITLE
Airflow Guide migration base PR

### PR DESCRIPTION
**Pull request by jwitz**
This PR will act as the main branch for the Airflow Guides migration process to Docs. Once it's ready, this PR will contain 100% of Airflow Guides that we want to migrate in their final form.

When merged, the new `Guides` (or `Learn`) tab will be available.

### CODE REVIEW

File: guides/airflow-azure-data-explorer.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+title: "Executing Azure Data Explorer Queries with Airflow"
+sidebar_label: "Executing Azure Data Explorer Queries with Airflow"
```
Comment (jwitz) [inline guides/airflow-azure-data-explorer.md:3]:
```suggestion
sidebar_label: "Azure Data Explorer"
```

File: sidebarsTutorials.js [hunk 2]
```diff
@@ -1,16 +1,85 @@
...
+          'subdags',
+          'task-groups',
+          'rerunning-dags',
+          'managing-dependencies',
+          'managing-airflow-code',
+          'templating',
+          'dynamically-generating-dags',
+      ],
+    },
+  ],
```
Comment (jwitz) [inline sidebarsTutorials.js:60]:
```suggestion
```

File: sidebarsTutorials.js [hunk 3]
```diff
@@ -1,16 +1,26 @@
...
- Create as many sidebars as you want.
- */
-
 module.exports = {
   tutorials: [
-    'overview',
+    'overview'
     'get-started-with-airflow',
     'astro-python-sdk',
+    'execute-notebooks',
```
Comment (jwitz) [inline sidebarsTutorials.js:6]:
```suggestion
```

File: sidebarsTutorials.js [hunk 4]
```diff
@@ -1,16 +1,25 @@
...
- - render a sidebar for each doc of that group
- - provide next/previous navigation
- The sidebars can be generated from the filesystem, or explicitly defined here.
- Create as many sidebars as you want.
- */
-
 module.exports = {
   tutorials: [
-    'overview',
+    'overview'
```
Comment (jwitz) [inline sidebarsTutorials.js:3]:
```suggestion
    'overview',
```

File: sidebarsTutorials.js [hunk 5]
```diff
@@ -1,16 +1,25 @@
...
+        'airflow-azure-data-explorer',
+        'airflow-openlineage',
+        'airflow-dbt',
+        'airflow-databricks',
+        'airflow-sagemaker',
+        'airflow-redshift',
+        'airflow-great-expectations',
+        'airflow-talend-integration',
+        'airflow-sql-data-quality-tutorial',
+        'astro-python-sdk-etl',
```
Comment (jwitz) [inline sidebarsTutorials.js:20]:
```suggestion
```

File: concepts/airflow-sql-data-quality.md [hunk 6]
```diff
@@ -0,0 +1,262 @@
+---
+title: "Airflow data quality checks with SQL Operators"
+sidebar_label: "Airflow data quality checks with SQL Operators"
```
Comment (jwitz) [inline concepts/airflow-sql-data-quality.md:3]:
```suggestion
sidebar_label: "SQL operators"
```

**Comment by kwcanuck** [comment]
@jwitz I wasn't sure where best to document issues with the new landing pages, so I'm doing it in this PR. It's probably premature, but I thought it worth documenting. 

On one page, we're using _Docs_:

image

On another, we're using _Documentation_:

image

We should use one term consistently. I recommend using _Documentation_.

On some pages, we're applying the trademark symbol, and on others we are not:

image

image

We should apply trademarks and attributions consistently. I recommend removing these symbols as our blanket attribution statement should cover trademarks owned by others. 

_Documentation_ is missing from the Astro CLI heading:

image

_Documentation_ should be added to make the heading consistent with the other landing page titles.

We should avoid the possessive form of our company name whenever possible:

image

Capitalization of titles is inconsistent:

image

image

All headings should be reviewed to make sure they are consistent.

As discussed previously, the text in some tiles is not wrapping:

image

We'll need to make sure we remove or update these tiles:

image

Gerunds are being used inconsistently:

image

image

I recommend removing the gerunds. Note also the capitalization issue identified previously.

We have a _What is Astro?_ text box on the Software landing page:

image

Should this be removed?

Selecting _Docs_ on the navigation bar opens the Astro landing page. Should it open the Home page?

The introductory text on the Home page should be updated as it currently discusses Astro. The introductory text should be more generic and discuss the _entire_ documentation suite. 

A tile for Astro should be added to the Home page.

We should try and avoid using terms such as _easy_ or _easiest_. From the Google developer style guide:

image

Watch for missing serial commas:

image

We shouldn't be using ampersands in titles or content:

image

**Pull request merged**